gkeys/etc/gkeys.conf

```diff
@@ -1,11 +1,18 @@
 # Gentoo-keys configuration file
 #
 
+[login_gentoo]
+login_email = gentoo
+passwd = gentoo_passwd
+server = smtp.gmail.com
+
```

Comment:
this is finally getting there... but, you didn't read my comments correctly.

Pleas put all email config's in a NEW file.  this new file can be stack loaded with any other config files at startup.  You just have to tell it to load it if args.email is set.